src/components/Practice/Mechanics3.jsx

```
@@ -363,60 +450,76 @@ const Mechanics2 = ({
+                        ),
+                        fontWeight: 600,
+                        fontSize: "30px", // Responsive font size
+                        fontFamily: "Quicksand",
+                      }}
+                    >
+                      {elem?.text}
+                    </span>
+                  </Box>
+                )
+            )}
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)